server/sonar-db-migration/src/main/java/org/sonar/server/platform/db/migration/version/v62/PopulateOrganizationUuidOfGroups.java

```diff
@@ -49,10 +49,10 @@ public void execute(Context context) throws SQLException {
     massUpdate.update("update groups set organization_uuid=?, updated_at=? where id=?");
     massUpdate.rowPluralName("groups");
     massUpdate.execute((row, update) -> {
-      long groupId = row.getLong(1);
+      int groupId = row.getInt(1);
```

Comment:
and since you're at it, resource id also should be an integer (and the integer max value is going to kick us in the n.ts much faster on this one than on users or groups)